impl/src/main/java/com/sun/faces/config/manager/tasks/ParseConfigResourceToDOMTask.java

```diff
@@ -78,6 +77,7 @@
 
     private static final String JAVAEE_SCHEMA_LEGACY_DEFAULT_NS = "http://java.sun.com/xml/ns/javaee";
     private static final String JAVAEE_SCHEMA_DEFAULT_NS = "http://xmlns.jcp.org/xml/ns/javaee";
+    private static final String JAKARTAEE_SCHEMA_DEFAULT_NS="https://jakarta.ee/xml/ns/jakartaee";
```

Comment:
Almost too tiny to remark, but the coding standard asks for a space left and right of the assignment,